src/server/coprocessor/endpoint.rs

```diff
@@ -803,6 +806,8 @@ impl SelectContextCore {
             }
             if col.get_pk_handle() {
                 box_try!(datum::encode_to(chunk.mut_rows_data(), &[get_pk(col, h)], false));
+            } else if col.has_default_val() {
```

Comment:
No for the current implementation.
 If the row doesn't contain the column, and it has default value it's not in public state.